eCommerce platforms/Adobe Magento/src/design/frontend/base/default/layout/dfp.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Microsoft Dynamics 365 Fraud Protection module for Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Microsoft
+ * @package    Dynamics 365 Fraud Protection
+ * @copyright  Copyright (c) Microsoft Corporation. (https://www.microsoft.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+-->
+
+<layout version="0.1.0">
+    <checkout_onepage_index translate="label">
+        <reference name="before_body_end">
+            <block type="dfp/index" name="dfp_index" template="dfp/fingerprinting.phtml"/>
+        </reference>	
+    </checkout_onepage_index>
+    <customer_account_edit translate="label">
+        <reference name="my.account.wrapper">
+            <block type="customer/form_edit" name="customer_edit" template="dfp/customer/form/edit.phtml"/>
+        </reference>
+    </customer_account_edit>   
+    <checkout_onepage_paymentmethod>        
+        <block type="checkout/onepage_payment_methods" name="root" output="toHtml" template="dfp/onepage/payment/methods.phtml">
```

Comment:
nit: other <block /> elements are self-closing but this one isn't

Reply:
Thanks, fixed.